pubspec.yaml

```diff
@@ -59,7 +59,7 @@ dependencies:
   jiffy: ^5.0.0
 
   # package_info_plus provides access to app specific information such as build number & version.
-  package_info_plus: ^3.0.2
+  package_info_plus: ^3.0.3
```

Comment:
This fixes a compilation error related to the recent changes to `Struct`s in dart:ffi.